bin/test-run

```
@@ -59,15 +76,11 @@ function run_upgrade_test() {
     local stable_namespace="$1"
     local stable_version=$(curl -s https://versioncheck.linkerd.io/version.json | grep -o "stable-[0-9]*.[0-9]*.[0-9]*")
 
-    printf "Installing release [%s] namespace [%s]\n" "$stable_version" "$stable_namespace"
     install_stable $stable_namespace
-
-    printf "Upgrading release [%s] to [%s]\n" "$stable_version" "$linkerd_version"
-    run_test "$test_directory/install_test.go" --upgrade-from-version=$stable_version --linkerd-namespace=$stable_namespace
+    run_test "$test_directory/install_test.go" -failfast --upgrade-from-version=$stable_version --linkerd-namespace=$stable_namespace
```

Fail fast for upgrade test. No point of continuing.